main.go

```diff
@@ -582,9 +604,23 @@ func (p *gateway) scheduleStatCollection(ctx context.Context, scheduler *sfxclie
 	return finishedContext, cancelFunc
 }
 
-func (p *gateway) setupForwardersAndListeners(ctx context.Context, loader *config.Loader, loadedConfig *config.GatewayConfig, logger log.Logger, scheduler *sfxclient.Scheduler) (signalfx.Sink, map[string]http.Handler, error) {
+func (p *gateway) scheduleStatCollectionWithDefaultDelay(ctx context.Context, scheduler *sfxclient.Scheduler, loadedConfig *config.GatewayConfig, multiplexer signalfx.Sink) (context.Context, context.CancelFunc) {
+	// We still want to schedule stat collection so people can debug the server if they want
+	scheduler.Sink = dpsink.Discard
```

Comment:
this is too much of a copy pasta of the other one. you don't need to set the sink to discard, it should always be the mux, and you'll never be skipping internal metrics stat keeping.